src/App.js

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
+import SidebarNav from './components/HamburgerMenu/SidebarNav';
```

Comment:
I don't believe you will be using the SidebarNav inside App.js, so after testing your code, clean up your App.js and revert it back to normal unless you are connecting all the components